src/pages/items/ItemsPage.tsx

```diff
@@ -186,14 +145,13 @@ const ItemsPage: React.FC = () => {
                             <CategoryTitle
```

Comment:
I think we should also rename this component to CategoryTitleEditable or CategoryHeadingEditable